pipelines/docker/kabanero-utils/scripts/enforce_stack_policy.sh

```diff
@@ -242,12 +231,7 @@
         fi
 
         # Get the target sha256 digest from the image registry. Use the proper credentials depending on what was passed to us
-        echo "$INFO RECONCILED_STACK_IMAGE_REGISTRY_HOST = $IMAGE_REGISTRY_HOST"
-        if [ -z "$IMAGE_REGISTRY_PASSWORD" ] || [ -z "$IMAGE_REGISTRY_USERNAME" ]; then
-           TARGET_DIGEST=$( skopeo inspect docker://"$IMAGE_REGISTRY_HOST"/"$PROJECT"/"$STACK_NAME":"$VERSION" | jq '.Digest' )
-        else
-           TARGET_DIGEST=$( skopeo inspect --creds="$IMAGE_REGISTRY_USERNAME":"$IMAGE_REGISTRY_PASSWORD" docker://"$IMAGE_REGISTRY_HOST"/"$PROJECT"/"$STACK_NAME":"$VERSION" | jq '.Digest' )
-        fi
+        TARGET_DIGEST=$( skopeo inspect docker://"$IMAGE_REGISTRY_HOST"/"$PROJECT"/"$STACK_NAME":"$VERSION" | jq '.Digest' )
```

Comment:
How would this now work for private registries? Do the secrets somehow get inherited by the scripts?

Reply:
yes , when i tested with artifactory url  in past, I just created a secret via tekton dashboard and push and pull worked with internal kubernetes magic or tekton dashboard magic.